### DISCUSSION TITLE
State of Remix + Native?

### DISCUSSION THREAD

**Discussion by jochenstu** [remix-run/remix#5275] [General]
It seems pretty obvious, that native apps are not going anywhere. How does Remix position itself here? I have seen a few attempts to do something around here, but it does seem like nothing really caught on and most stuff has not been updated for a while now. What is the recommended, simplest way to go about Web + Native with Remix as of Jan 2023?

**Comment by gut4** [comment]
Remix electron stack looks interesting https://github.com/itsMapleLeaf/remix-electron/tree/v2

**Reply by jochenstu** [reply]
Pretty cool. How to go about native on mobile?
